pkg/oc/admin/migrate/migrator.go

```diff
@@ -49,6 +50,11 @@ func AlwaysRequiresMigration(_ *resource.Info) (Reporter, error) {
 	return ReporterBool(true), nil
 }
 
+// TimeStampNow returns the current time in the same format as glog
+func TimeStampNow() string {
```

Comment:
make this private and add a test, otherwise LGTM

Reply:
Exactly what should I add a test for?

Reply:
Add a test for your change.